crates/extricrate/src/lib.rs

```diff
@@ -62,11 +152,39 @@ pub mod dependencies {
 
     #[cfg(test)]
     mod tests {
+        use std::{collections::HashMap, path::Path};
+
         use pretty_assertions::assert_eq;
 
+        use crate::dependencies::{
+            Extent, File, ModuleName, NormalizedUseStatement, Position, UseStatement,
+            UseStatementType, list_use_statements,
+        };
+
         #[test]
-        fn sample_test() {
-            assert_eq!(1, 1);
+        fn get_simple_dependency() {
+            let test_project =
+                Path::new(env!("CARGO_MANIFEST_DIR")).join("tests/fixtures/simple/main.rs");
+            let res = list_use_statements(&test_project).expect("Failed to list statements");
+            let mut expected = HashMap::new();
+            expected.insert(
+                File("src/main.rs".to_owned()),
```

Comment:
I think you would have hit that sooner or later, but your current setup doesn't include a src folder (and I think it should, for testing the directory walking logic that you haven't written, but that can come later)